app/src/main/java/org/schabi/newpipe/local/subscription/item/FeedImportExportItem.kt

```diff
@@ -0,0 +1,111 @@
+package org.schabi.newpipe.local.subscription.item
+
+import android.graphics.Color
+import android.graphics.PorterDuff
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.annotation.DrawableRes
+import com.xwray.groupie.kotlinandroidextensions.GroupieViewHolder
+import com.xwray.groupie.kotlinandroidextensions.Item
+import kotlinx.android.synthetic.main.feed_import_export_group.*
+import org.schabi.newpipe.R
+import org.schabi.newpipe.extractor.NewPipe
+import org.schabi.newpipe.extractor.exceptions.ExtractionException
+import org.schabi.newpipe.util.AnimationUtils
+import org.schabi.newpipe.util.ServiceHelper
+import org.schabi.newpipe.util.ThemeHelper
+import org.schabi.newpipe.views.CollapsibleView
+
+
+class FeedImportExportItem(
+        val onImportPreviousSelected: () -> Unit,
+        val onImportFromServiceSelected: (Int) -> Unit,
+        val onExportSelected: () -> Unit
+) : Item() {
+    companion object {
+        const val EXPAND_LIST = 123
+    }
+
+    override fun bind(viewHolder: GroupieViewHolder, position: Int, payloads: MutableList<Any>) {
+        if (payloads.contains(EXPAND_LIST)) {
+            viewHolder.import_export_options.expand()
+            return
+        }
+
+        super.bind(viewHolder, position, payloads)
+    }
+
+    override fun getLayout(): Int = R.layout.feed_import_export_group
+
+    override fun bind(viewHolder: GroupieViewHolder, position: Int) {
+        if (viewHolder.import_from_options.childCount == 0) setupImportFromItems(viewHolder.import_from_options)
+        if (viewHolder.export_to_options.childCount == 0) setupExportToItems(viewHolder.export_to_options)
+
+        expandIconListener?.let { viewHolder.import_export_options.removeListener(it) }
+        expandIconListener = CollapsibleView.StateListener { newState ->
+            AnimationUtils.animateRotation(viewHolder.import_export_expand_icon,
+                    250, if (newState == CollapsibleView.COLLAPSED) 0 else 180)
```

Comment:
If this `250` is related to the one in `app/src/main/java/org/schabi/newpipe/local/subscription/dialog/FeedGroupDialog.kt` then create a unique constant, otherwise you can leave it here without giving it a name since it is only used once.

Reply:
See [my previous answer](https://github.com/TeamNewPipe/NewPipe/pull/2309#discussion_r371625549) about it.